stable/kong/templates/prometheus-plugin.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if and .Values.ingressController.enabled .Values.ingressController.metrics.enabled }}
+apiVersion: configuration.konghq.com/v1
+kind: KongPlugin
+metadata:
+  name: prometheus
```

Comment:
Please include the release name in the name of this KongPlugin to avoid any collisions.